ipaserver/install/replication.py

```diff
@@ -1547,6 +1548,9 @@ def disable_agreement(self, hostname):
         Disable the replication agreement to hostname.
         """
         entry = self.get_replication_agreement(hostname)
+        if not entry:
+            raise errors.NotFound(
+                reason=_("Replication agreement for %s not found" % hostname))
```

Comment:
This violates rules of how to write translation strings

https://www.freeipa.org/page/Python_Coding_Style#Python_Internationalized_.28i18n.29_Strings

Also installers haven't been internationalized, only plugins/webUI/CLI. So I'd check if translation works for installers or additional changes may be needed.

Reply:
To be exact:
* is  better for translators to always have keyword substitutions (even just for one replacement)
* it should be consistent
* if you need to add additional substitution, you have to rewrite it anyway to keywords
* I think this will not work `_("%s" % test)`. You need `_("%s") % test`

Also please remove this substring `and it has more than one format substitution` from wiki